src/main/webapp/WEB-INF/tags/helpPage.tag

```diff
@@ -31,9 +31,7 @@
     <div id="footerComponent" class="container-fluid">
         <div class="container">
             <div class="row">
-                <div class="col-md-2">
-                    <span>[<a href="/index.jsp">TEAMMATES</a>]</span>
-                </div>
+                <t:navBarBrand/>
```

Comment:
Wrong place - you're editing the footer here, not the navbar. Also, same as above - only replace the existing `<a>` tag and its contents.